_posts/2021-09-13-changelog-155.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #155"
+date: 2021-09-13 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#7782] Enable fetching actual info about stdlib packages like dependencies, [edition](https://doc.rust-lang.org/edition-guide/editions/index.html), Cargo [features](https://doc.rust-lang.org/cargo/reference/features.html) by default. It should help the plugin better understand stdlib structure, for example, provide proper completion and navigation for items defined in `std::os` module. In case of any issue, you can turn this feature off via `org.rust.cargo.fetch.actual.stdlib.metadata` [experimental feature](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-faq.html#experimental-features)
+
+* [#7772] Show warning/errors during fetching stdlib info in [Sync View](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-cargo-tool-window.html)
+
+* [#7777] Enable by default code completion ranking for Rust based on machine learning. Now the completion suggestions are sorted by their relevance based on the rules learned from data we have gathered anonymously during our EAPs. Note that we have not collected any source code, only information about your interactions with the code completion UI. In case of any issues, you can disable the new ranking model via `Editor | General | Code Completion | Machine Learning-Assisted Completion` settings
+
+* [#7769], [#7751] Correctly parse `unsafe` keyword in extern and module blocks, annotate them as errors
+  <img src="/assets/posts/changelog-155/unsafe-extern-mod.png" alt="Parse and annotate unsafe in extern and module blocks" width="700px">
+
+* [#7734] Introduce new icons for macro items, providing `macro_rules!` and `macros 2.0` items with distinctive ones
+  <img src="/assets/posts/changelog-155/new-macro-icons.png" alt="New macro items icons" width="700px">
+
+* [#7663] Support `Move Element Left/Right` action for predicates in `where` clause
+  {% include gif-img.html path="/assets/posts/changelog-155/where-move" w="700px" %}
+
+* [#7757] Parse and annotate `const_generics_defaults` compiler feature
+
+* [#3790] Deprecated item inspection now correctly handles items that will be deprecated in a future version (by [@mibac138])
+
+## Fixes
+
+* [#7795] Improve parsing of const generic arguments for `const_generics_defaults` compiler feature, provide inspection and quick-fix for expressions that must be enclosed
```

Comment:
Let's drop it since it's not included in the corresponding release. Don't forget to drop the corresponding link to PR below as well